code/modules/recycling/conveyor2.dm

```diff
@@ -218,6 +221,8 @@
 	var/list/conveyors		// the list of converyors that are controlled by this switch
 	anchored = 1
 
+/obj/machinery/conveyor_switch/two_way_on
+	position = 1
```

Comment:
I am not sure why we would want default on conveyors/conveyor switches

Reply:
Mostly so I didnt have to make a var edit version, I wanted one that would be default on